src/main/java/net/minecraftforge/common/ForgeConfigSpec.java

```diff
@@ -249,6 +249,24 @@ else if (dryRun)
         return count;
     }
 
+    private boolean stringsMatchIgnoringNewlines(Object obj1, Object obj2)
```

Comment:
I'll just make a note here for future reference:
This function can be passed null, for eg. when the config is first being validated.
Calling replaceAll on null causes an immediate hard crash, so i had to add instanceof and length checks to ensure it's even valid for replacement. Without it, this just causes bigger issues.

Someone asked, so i thought i'd make a note here.

Reply:
Since it may be passed null, why not mark the parameters `@Nullable`?

Reply:
Good point! One moment.